harness/determined/common/experimental/trial.py

```diff
@@ -16,9 +17,13 @@ class is obtained via :class:`determined.experimental.Determined`, the
             master URL is automatically passed into this constructor.
```

Comment:
another instance where we need to remove the Arguments from the docstring; users should not instantiate this on their own.